legal-api/src/legal_api/models/business.py

```diff
@@ -414,6 +416,24 @@ def good_standing(self):
                 return date_cutoff.replace(tzinfo=pytz.UTC) > datetime.utcnow()
         return True
 
+    @property
+    def in_dissolution(self):
+        """Return true if in dissolution, otherwise false."""
+        # check a business has a batch_processing entry that matches business_id and status is not COMPLETED
+        find_in_batch_processing = db.session.query(BatchProcessing).filter(
+            BatchProcessing.business_id == self.id,
+            BatchProcessing.status != BatchProcessing.BatchProcessingStatus.COMPLETED,
+            BatchProcessing.status != BatchProcessing.BatchProcessingStatus.WITHDRAWN,
```

Comment:
Also, this can be simplified too 😺 
```
BatchProcessing.status.notin_([BatchProcessing.BatchProcessingStatus.COMPLETED.value,
                                                  BatchProcessing.BatchProcessingStatus.WITHDRAWN.value])
```